app/models/katello/candlepin/repository_mapper.rb

```diff
@@ -106,7 +106,11 @@ def katello_content_type
       end
 
       def unprotected?
-        kickstart? || file?
+        kickstart? || file? || suse?
+      end
+
+      def suse?
+        content.content_type == Repository::YUM_TYPE && path.downcase.include?("/suse")
```

Comment:
are we sure that all of the suse paths will match this check? do you think something like `path.downcase.match(/suse/)` might be better (or worse?)